lib/service/config.go

```diff
@@ -1,27 +1,56 @@
-	EnablePrometheus      bool   `envconfig:"ENABLE_PROMETHEUS" default:"false"`
-	PrometheusPort        int    `envconfig:"PROMETHEUS_PORT" default:"9092"`
-	WebhookUrl            string `envconfig:"WEBHOOK_URL"`
-	FeeReserve            bool   `envconfig:"FEE_RESERVE" default:"false"`
-	AllowAccountCreation  bool   `envconfig:"ALLOW_ACCOUNT_CREATION" default:"true"`
-	MaxReceiveAmount      int64  `envconfig:"MAX_RECEIVE_AMOUNT" default:"0"`
-	MaxSendAmount         int64  `envconfig:"MAX_SEND_AMOUNT" default:"0"`
-	MaxAccountBalance     int64  `envconfig:"MAX_ACCOUNT_BALANCE" default:"0"`
+	DatabaseUri           string        `envconfig:"DATABASE_URI" required:"true"`
+	SentryDSN             string        `envconfig:"SENTRY_DSN"`
+	LogFilePath           string        `envconfig:"LOG_FILE_PATH"`
+	JWTSecret             []byte        `envconfig:"JWT_SECRET" required:"true"`
+	JWTRefreshTokenExpiry int           `envconfig:"JWT_REFRESH_EXPIRY" default:"604800"` // in seconds, default 7 days
+	JWTAccessTokenExpiry  int           `envconfig:"JWT_ACCESS_EXPIRY" default:"172800"`  // in seconds, default 2 days
+	LNDAddress            string        `envconfig:"LND_ADDRESS" required:"true"`
+	LNDMacaroonHex        string        `envconfig:"LND_MACAROON_HEX" required:"true"`
+	LNDCertHex            string        `envconfig:"LND_CERT_HEX"`
+	CustomName            string        `envconfig:"CUSTOM_NAME"`
+	Host                  string        `envconfig:"HOST" default:"localhost:3000"`
+	Port                  int           `envconfig:"PORT" default:"3000"`
+	DefaultRateLimit      int           `envconfig:"DEFAULT_RATE_LIMIT" default:"10"`
+	StrictRateLimit       int           `envconfig:"STRICT_RATE_LIMIT" default:"10"`
+	BurstRateLimit        int           `envconfig:"BURST_RATE_LIMIT" default:"1"`
+	EnablePrometheus      bool          `envconfig:"ENABLE_PROMETHEUS" default:"false"`
+	PrometheusPort        int           `envconfig:"PROMETHEUS_PORT" default:"9092"`
+	WebhookUrl            string        `envconfig:"WEBHOOK_URL"`
+	FeeReserve            bool          `envconfig:"FEE_RESERVE" default:"false"`
+	AllowAccountCreation  bool          `envconfig:"ALLOW_ACCOUNT_CREATION" default:"true"`
+	MaxReceiveAmount      int64         `envconfig:"MAX_RECEIVE_AMOUNT" default:"0"`
+	MaxSendAmount         int64         `envconfig:"MAX_SEND_AMOUNT" default:"0"`
+	MaxAccountBalance     int64         `envconfig:"MAX_ACCOUNT_BALANCE" default:"0"`
+	BrandingTitle         string        `envconfig:"BRANDING_TITLE" default:"LNDhub.go - Alby Lightning"`
+	BrandingDesc          string        `envconfig:"BRANDING_DESC" default:"Alby server for the Lightning Network"`
+	BrandingUrl           string        `envconfig:"BRANDING_URL" default:"https://ln.getalby.com"`
+	BrandingLogo          string        `envconfig:"BRANDING_LOGO" default:""`
+	BrandingFavicon       string        `envconfig:"BRANDING_FAVICON" default:"../static/img/favicon.png"`
```

Comment:
that actually should be /static/img/favicon.png :) - do we even need a favicon? 

customizing those is actually a bit of a problem because right now everything is embedded in the compiled lndhub executable. 
When those are customized those would need to be served from somewhere else.
To configure those a full URL would need to be used. 

wondering if we could simplify this? 

For example: 
IF some config option to a directory is set. then we serve the static files in that directory and use the index.html in that folder as template. 

just thinking out load. 

but +1 for having this configurable! when people run their own lndhub the Alby branding is confusing.

Reply:
> To configure those a full URL would need to be used.

You are right. I thought that `data:image/png;base64,....` would work just fine, but I run into https://github.com/golang/go/issues/20536 which forbids that for <img src="{{ ... }}">

> IF some config option to a directory is set. then we serve the static files in that directory and use the index.html in that folder as template.

What I like about my approach is that it still could work in a deployment where `lndhub` binary does not have an access to the filesystem (just sockets). If that is out of scope, we can can use your solution and create an env arguments which overrides the whole`templates/index.html`.

Let me know what you think and I'll update the PR.